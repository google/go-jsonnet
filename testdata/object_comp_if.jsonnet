{ [x]: 42 for x in ["a", "b", "bb", "c"] if x[0] == "b" }
