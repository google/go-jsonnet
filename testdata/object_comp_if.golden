{
   "b": 42,
   "bb": 42
}
